language_server/src/lsp/feature_language_features.h

```diff
@@ -15,6 +15,8 @@
 #ifndef HLASMPLUGIN_LANGUAGESERVER_FEATURE_LANGUAGEFEATURES_H
 #define HLASMPLUGIN_LANGUAGESERVER_FEATURE_LANGUAGEFEATURES_H
 
+#include <string>
+#include <unordered_map>
```

Comment:
Seems like we're missing include for `nlohmann::json`.
Or it might be better to say we should consolidate the includes in .h and .cpp

Reply:
The feature.h contains include with forward declarations... Given its a base class, I think it is ok.